e2e-tests/playwright/utils/Common.ts

```diff
@@ -54,13 +54,50 @@ export class Common {
 
   async loginAsGithubUser() {
     await this.logintoGithub();
-    await this.page.goto('/');
+    await this.page.goto('/', { waitUntil: 'networkidle' });
     await this.page.waitForLoadState('load');
     await this.uiHelper.clickButton('Sign In');
     await this.checkAndReauthorizeGithubApp();
     await this.uiHelper.waitForSideBarVisible();
   }
 
+  async loginAsGithubUserWithRetry(retries: number = 3) {
```

Comment:
@gustavolira why do we need to retry for login? It should work on the first attempt.

Reply:
> @gustavolira why do we need to retry for login? It should work on the first attempt.

@subhashkhileri  it's because this is the most flaky test, if it works at the first attempt the retry loop will be stopped and the code will move forward